aptos/example/example_app/sources/minting.move

```diff
@@ -0,0 +1,102 @@
+module mint_nft::minting {
+    use std::signer;
+    use std::string::{Self, String};
+    use std::vector;
+
+    use aptos_framework::account;
+    use aptos_framework::event::EventHandle;
+    use aptos_framework::coin;
+    use aptos_framework::aptos_coin;
+
+    use pyth::pyth;
+    use pyth::price_identifier;
+    use pyth::i64;
+    use pyth::price;
+
+    use aptos_std::math64::pow;
+
+    use aptos_token::token::{Self, TokenDataId};
+
+    const APTOS_USD_PRICE_FEED_IDENTIFIER : vector<u8> = x"ca80ba6dc32e08d06f1aa886011eed1d77c77be9eb761cc10d72b7d0a2fd57a6"; // This is actually the ETH/USD while APT is not listed
+
+    // This event stores the receiver of the NFT and the TokenDataId of the NFT
+    struct TokenMintingEvent has drop, store {
+        token_receiver_address: address,
+        token_data_id: TokenDataId,
+    }
+
+    // This struct stores an NFT relevant information and the signer capability required to mint the NFT
+    struct CollectionTokenMinter has key {
+        token_data_id: TokenDataId,
+        token_minting_events: EventHandle<TokenMintingEvent>,
+        signer_cap: account::SignerCapability
+    }
+
+    /// Octas per aptos coin
+    const OCTAS_PER_APTOS: u64 = 100000000;
```

Comment:
It's a shame that this isn't in the stdlib.

Reply:
Yes